### ISSUE TITLE
Graph's state and values are lost while removing the  background grid lines

### ISSUE THREAD

**Issue by shivangT** [ToolJet/ToolJet#3249]
### What is the expected behaviour?
When we toggle it on the expected graph values should not be lost and the state should be maintained. The state should be reverted back by toggling on  the "show grid lines".

### What is the current behaviour?
The graph doesn't behave as expected. The state is not maintained and the values are lost. You can revert it back by toggling on the "Show axes".

### How to reproduce the issue?

1.  Visit this [link](https://app.tooljet.com/)
2. Now click on "Create New Application" 
3. Now drag-drop the graph widget from the widgets inventory on the right hand side
4. Now inspect the widget and on the right hand side you'll find the properties associated with the widget
5. scroll down and you'll find  options called "Show Axes" and "Show Grid Lines"
6.  Now toggle off show grid lines, it distorts the actual graph i.e the values of x-y axis is lost
7. You can revert the actual state only when you toggle on/off "Show Axes" and not the "Show grid lines" 

### Screenshots or Screencast

https://user-images.githubusercontent.com/32157641/172878855-21b6e1ce-0556-4055-a241-b6330badfba3.mov

**Comment by hrutik7** [comment]
can i work on this?

**Comment by shivangT** [comment]
Hello @hrutik7 ,
Is it clear enough to you?
We're looking to modify the content of this issue. We'll shortly get back to you.

**Comment by hrutik7** [comment]
Sure

**Comment by shivangT** [comment]
Hey @hrutik7 
I was hoping if you have gone through the code. Let me know if you're available to look into this.

Thanks

**Comment by hrutik7** [comment]
Yeah please assign this issue to me, I was working that Accordion issue I just solved that issue and want to work on this

**Comment by shivangT** [comment]
Hello @hrutik7 
Just reverted you on the other issue. You can read the following issue and let me know if you want me to assign this to you.
Have a good day

**Comment by shivangT** [comment]
Hey @hrutik7 
Great, assigning this issue to you. 
We'll look into your PR once our reviewers are available. You'll be notified, until then you can move ahead with this PR.
Thanks :)

**Comment by shivangT** [comment]
Hey @hrutik7 

Kindly respond to help us learn if you’re stuck with anything. This is a long due issue and we need to close it. 

Please confirm before i un-assign you and assign to someone else.

Thanks

**Comment by hrutik7** [comment]
hello @shivangT 
I've been busy in some other projects and tasks
for this time please un-assign this issue to me
I am sorry for the inconvenience as soon as I get time
I am looking for collaborating on tooljet once again 

thanks

**Comment by shivangT** [comment]
Sure, thanks for reverting on the same.

**Comment by mikebarr24** [comment]
Hey @shivangT. If this issue is still up for grabs, I'd like to have a go.
